### ISSUE TITLE
Vich TicketMessageWithAttachment.xml should be renamed to Entity.TicketMessageWithAttachment.xml

### ISSUE THREAD

**Issue by gomcodoctor** [hackzilla-project/TicketBundle#151]
Vich

```
TicketMessageWithAttachment.xml 
```
should be renamed to 

```
Entity.TicketMessageWithAttachment.xml
```

otherwise it give TicketMessageWithAttachment class do not exists error

**Comment by phansys** [comment]
Thank you so much for the report @gomcodoctor.
Could you please create a PR with the proposed fix?

**Comment by hackzilla** [comment]
I've pushed out a fix in 3.8.1
